gdk/aws_client_utils/S3ClientUtils.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from botocore.exceptions import ClientError
+
+
+class S3ClientUtils:
```

Comment:
Could we just call it as `S3Client` it will just be a wrapper around the boto s3 client. If we want to be a bit more accurate about what this really is we could call it `S3Adapter`